upb/pb/decoder.c

```diff
@@ -596,6 +599,14 @@ int32_t upb_pbdecoder_skipunknown(upb_pbdecoder *d, int32_t fieldnum,
     }
 
     if (d->top->groupnum >= 0) {
+      addunknown = (upb_addunknown_handlerfunc *)upb_handlers_gethandler(
```

Comment:
Can't you replace all this code with upb_sink_putunknown()?

Reply:
Done